### PR TITLE
Add template for GitHub issues

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vince@vincepri.com>

**What this PR does / why we need it**:
Adds bug/features templates for github issues.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by vincepri** [comment]
Flaky test?

/retest

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+[A clear and concise description of what you want to happen.]
+
+
+**Anything else you would like to add:**
+[Miscellaneous information that will assist in solving the issue.]
+
+
+**Environment:**
+
+- Cluster-api-provider-aws version:
```
Comment (roberthbailey) [inline .github/ISSUE_TEMPLATE/feature_request.md]: cluster-api version.

**Comment by roberthbailey** [comment]
Looks like one typo copying over from the aws provider repo. Can you fix that so we can get this merged? Thanks!

**Comment by vincepri** [comment]
Done!

**Pull request merged**